### ISSUE TITLE
Can't install 0.13 on Fedora 38

### ISSUE THREAD

**Issue by OperKH** [kopia/kopia#2997]
I have default installation from https://kopia.io/docs/installation/ "Linux installation using RPM (RedHat, CentOS, Fedora)" section.
Currently I'm on 0.12 version.

And I can't upgrade my system due to kopia signature error:
```
Invalid package file:

package kopia-0.13.0-1.x86_64 cannot be verified and repo Kopia is GPG enabled: /var/cache/PackageKit/38/metadata/Kopia-38-x86_64/packages/kopia-0.13.0.x86_64.rpm could not be verified. /var/cache/PackageKit/38/metadata/Kopia-38-x86_64/packages/kopia-0.13.0.x86_64.rpm: digest: SIGNATURE: NOT OK
```

**Comment by Nimon77** [comment]
Same here, maybe related to #2947

**Comment by jkowalski** [comment]
The original signing key has unfortunately expired just before 0.13 was released and an updated key is now available. New installs should be fine but upgrades from <0.13 will need to re-import the key:

```
rpm --import https://kopia.io/signing-key
```

**Comment by OperKH** [comment]
Run:
```sh
rpm --import https://kopia.io/signing-key
dnf upgrade --refresh
```
Output:
>Kopia                                                                                                                                                         8.4 kB/s | 1.6 kB     00:00    
GPG key at https://kopia.io/signing-key (0xFD56A556) is already installed
The GPG keys listed for the "Kopia" repository are already installed but they are not correct for this package.
Check that the correct key URLs are configured for this repository.. Failing package is: kopia-0.13.0-1.x86_64
 GPG Keys are configured as: https://kopia.io/signing-key
Public key for kopia-ui-0.13.0.x86_64.rpm is not trusted. Failing package is: kopia-ui-0.13.0-1.x86_64
 GPG Keys are configured as: https://kopia.io/signing-key
The downloaded packages were saved in cache until the next successful transaction.
You can remove cached packages by executing 'dnf clean packages'.
Error: GPG check FAILED

Tried to remove and install it again:
```sh
rpm --erase --allmatches gpg-pubkey-fd56a556-6078ec06
rpm --import https://kopia.io/signing-key
dnf upgrade --refresh
```

Output:
> error: rpmdbNextIterator: skipping h#    3574 
Header V4 RSA/SHA512 Signature, key ID fd56a556: BAD
Header SHA256 digest: OK
error: rpmdbNextIterator: skipping h#    3574 
Header V4 RSA/SHA512 Signature, key ID fd56a556: BAD
Header SHA256 digest: OK

**Comment by OperKH** [comment]
Looks like it was rpm issue.

Today Fedora released rpm 4.18.1 and I successfully updated kopia.

Here is rpm 4.18.1 changelog https://rpm.org/wiki/Releases/4.18.1 there is some OpenPGP fixes.
